### PR TITLE
Revert to revised list of search time synonyms

### PR DESCRIPTION
**Pull request by MatMoore**
This PR moves the B-variant back to a clean slate- it should work the same as A, but with some outdated synonyms removed.

We're now only planning to change a subset of these synonyms for this sprint, so we will reintroduce the changes in future PRs after more testing.

See individual commits for more details.

Trello: https://trello.com/c/9VF3Rw5V/451-test-synonyms-with-healthcheck-and-search-performance-explorer

### CODE REVIEW

File: config/schema/synonyms.yml [hunk 1]
```diff
@@ -212,108 +216,158 @@
...
+- search: reg => registration
+- search: ref number => ref number, reference number
+- search: tacho, tacho graph, taco, taco graph => tachograph
 # eg unique tax reference, unique tax code
-- both: unique tax => unique taxpayer
+- search: unique tax => unique taxpayer
+
 # Correcting mistakes
-# "advise => advise, advice",
+# advise => advise, advice
```
Comment (dwhenry) [inline config/schema/synonyms.yml:370]: this can most likely be removed

File: config/schema/synonyms.yml [hunk 2]
```diff
@@ -570,7 +632,7 @@
 - search: piper j3 => piper j3, piper j3c, piper j3f, piper j3l
 - search: piper j4 => piper j4, piper j4a
 - search: piper j5 => piper j5, piper j5a
-# "l4 => l4a, l4b, l4h, l4j", # affects other L4 content
+# l4 => l4a, l4b, l4h, l4j # affects other L4 content
```
Comment (dwhenry) [inline config/schema/synonyms.yml:635]: This one as well

**Review by dwhenry — APPROVED** [review]
All looks good I think

Might be worth getting another pair of eyes as well.

**Comment by MatMoore** [comment]
Closing this as the important commits (12915b3 and 586a52b) will be cherry picked into Suzanne's branch.

**Pull request closed without merging**